library/src/main/java/com/schibsted/spain/barista/BetterScrollToAction.java

```diff
@@ -0,0 +1,46 @@
+package com.schibsted.spain.barista;
+
+import android.support.test.espresso.UiController;
+import android.support.test.espresso.ViewAction;
+import android.support.test.espresso.action.ScrollToAction;
+import android.support.test.espresso.matcher.ViewMatchers;
+import android.support.v4.widget.NestedScrollView;
+import android.view.View;
+import android.widget.HorizontalScrollView;
+import android.widget.ScrollView;
+import org.hamcrest.Matcher;
+
+import static android.support.test.espresso.matcher.ViewMatchers.isAssignableFrom;
+import static android.support.test.espresso.matcher.ViewMatchers.isDescendantOfA;
+import static android.support.test.espresso.matcher.ViewMatchers.withEffectiveVisibility;
+import static org.hamcrest.Matchers.allOf;
+import static org.hamcrest.Matchers.anyOf;
+
+public class BetterScrollToAction implements ViewAction {
+  private final ScrollToAction scrollToAction;
+
+  public static BetterScrollToAction scrollTo() {
+    return new BetterScrollToAction();
+  }
+
+  private BetterScrollToAction() {
+    scrollToAction = new ScrollToAction();
+  }
+
+  @SuppressWarnings("unchecked")
+  @Override
+  public Matcher<View> getConstraints() {
+    return allOf(withEffectiveVisibility(ViewMatchers.Visibility.VISIBLE), isDescendantOfA(anyOf(
+        isAssignableFrom(ScrollView.class), isAssignableFrom(HorizontalScrollView.class), isAssignableFrom(NestedScrollView.class))));
```

Comment:
Maybe you're seeing a different version than mine?
Here's the code in the repo: https://android.googlesource.com/platform/frameworks/testing/+/android-support-test/espresso/core/src/main/java/android/support/test/espresso/action/ScrollToAction.java#53